Jenkinsfile-insertions

```diff
@@ -84,5 +84,14 @@ pipeline{
 				}
 			}
 		}
-    	}
+  }
+}
+
+// Utility function that checks upstream builds of this project were successfully built.
+def checkUpstreamBuildsSucceeded(String stepName) {
+	def statusUrl = httpRequest authentication: 'jenkinsKey', url: "${env.JENKINS_JOB_URL}/job/${env.RELEASE_NUMBER}/job/$stepName/lastBuild/api/json"
+	def statusJson = new JsonSlurper().parseText(statusUrl.getContent())
+	if(statusJson['result'] != "SUCCESS"){
+		error("Most recent $stepName build status: " + statusJson['result'])
+	}
 }
```

Comment:
This is really good :).  I was wondering since this check is done in many release steps for the Jenkins pipeline, is there a way to make it a utility function at higher level so that it can be imported and reused across steps?  

If so, it'd be good to make a GitHub issue to do that for a new PR.

Reply:
To the best of my knowledge, we'd have to do it through a bash script, similar to the 'confirmJenkinsConfigs' in the scripts folder. We'd have to supply it the URL and authentication information for accessing the Jenkins API. I don't see why that wouldn't be possible so I'll create an issue

Reply:
Created one in data-release-pipeline repo: https://github.com/reactome/data-release-pipeline/issues/100